src/main/java/nl/beehivejava/scriptj/parser/PluginInformationParser.java

```diff
@@ -0,0 +1,15 @@
+package nl.beehivejava.scriptj.parser;
+
+import nl.beehivejava.scriptj.PluginInformation;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * @author Lesley
+ */
+public interface PluginInformationParser {
+
+    PluginInformation parse(InputStream is) throws IOException;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)